From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sat, 11 Jan 2025 21:20:42 +0100
Subject: arm64: dts: rk3588-orangepi-5-plus: Lower SPI frequency to 50 MHz

100 MHz doesn't work well. Read transactions seem to return 0 byte
at the beginning with 100 MHz clock.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
@@ -343,6 +343,11 @@ &recovery_button {
 &sfc {
 	pinctrl-names = "default";
 	pinctrl-0 = <&fspim1_pins>;
+
+};
+
+&spi_flash {
+	spi-max-frequency = <50000000>;
 };
 
 &u2phy1_otg {
-- 
Armbian

